nengo/connection.py

```diff
@@ -347,6 +346,9 @@ def size_out(self):
 
     @property
     def _label(self):
```

Comment:
Maybe we should call this something other than `_label` now? like `_str`, or `_tag`, or something.